packages/application-components/src/test-utils/test-utils.tsx

```diff
@@ -52,9 +58,11 @@ const customRender = (
         .key(projectKey)
         .buildGraphql<TProjectGraphql>()}
     >
-      <IntlProvider locale={locale}>
-        <Router history={history}>{node}</Router>
-      </IntlProvider>
+      <ApolloProvider client={apolloClient}>
```

Comment:
Wrapping the component rendered in an `ApolloProvider` so `useMcQuery` hooks and such can work properly.